karapace.auth.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Maybe this file could be under `tests` and serve as a template from there for final auth database files. Reduces the risk of running Karapace with default auth database.